### ISSUE TITLE
Unable to find out the reference to 'fitted_tfidf_to_use.pickle' 

### ISSUE THREAD

**Issue by vaisakh-kp** [xianjinseow92/projects#1]
I tried setting up the consumer complaint classification flask app in local machine. While im unable to find out the 'fitted_tfidf_to_use.pickle' you used inside the predictor_api.py file. I tried the same method you applied there to generate the Tfidf object. But getting a file of size more than 2 GB!. How you managed to output the size arround 1 mb?

**Comment by siddharthshenoy** [comment]
hey im trying to set up the same project but on a complete different dataset and i see that you were able to get fitted_tfidf_to_use.pickle and you share the code with me as even i was in a hiccup in understanding how to do it

**Comment by vaisakh-kp** [comment]
The idea is that you have to use the `fit` function on tfidf value to get the `fitted_tfidf_to_use.pickle`  referred in the program. 

`X, y = df_category_and_content.REVIEW_CONTENT, df_category_and_content.CATEGORY`
`X_train_val, X_test, y_train_val, y_test = train_test_split(X, y, 
                                                            test_size=0.2,
                                                            stratify=y,
                                                            random_state=seed)`
`tfidf = TfidfVectorizer(sublinear_tf=True,  min_df=5,  stop_words='english')`
`fitted_tfidf_to_use = tfidf.fit(X_train_val)`

`with open('fitted_tfidf_to_use.pickle', 'wb') as to_write:`
    `pickle.dump(fitted_tfidf_to_use, to_write)`
    
    
 You can use the `fitted_tfidf_to_use.pickle` and `logit_ebay.pickle` further on your code.

**Comment by siddharthshenoy** [comment]
Thank you soo much I'll try to execute this on my dataset and see

**Comment by siddharthshenoy** [comment]
thanks dude your code works like a charm :)
